src/core_plugins/metrics/public/components/__tests__/add_delete_buttons.js

```diff
@@ -0,0 +1,67 @@
+// import React from 'react';
+// import { expect } from 'chai';
+// import { shallow } from 'enzyme';
+// import sinon from 'sinon';
+// import AddDeleteButtons from '../add_delete_buttons';
+// import Tooltip from '../tooltip';
+
+// describe('<AddDeleteButtons />', () => {
```

Comment:
Both giving LGTMs and merging while tests are commented out? Several -1s from me. I'm hoping there is an issue created already to fix this? If we can't get tests to run we have to either fix them or delete them.